stages/dotnet-package.yml

```diff
@@ -3,6 +3,11 @@ parameters:
     type: string
     default: Ubuntu-latest
 
+# Debug
+  - name: buildConfiguration
+    type: string
+    default: Debug
```

Comment:
Should the default not be release? And then the whoever will choose whether it should be Debug?

If this is kept as Debug and merged to master all packages will be built with a debug config.